core/variables.tf

```diff
@@ -25,3 +25,28 @@ variable "secret_key_base" {
   type        = string
   sensitive   = true
 }
+
+variable "rds_database_name" {
+  description = "RDS database name"
+  type        = string
+}
+
+variable "rds_username" {
+  description = "RDS username"
+  type        = string
+}
+
+variable "rds_password" {
```

Comment:
password should be marked as sensitive, otherwise it can be printed in logs

Reply:
Ah, my bad, updated them all in [549e484](https://github.com/nvminhtue/tyson_devops_ic/pull/27/commits/549e4847af5d883043e7b6c163b3b274ac9518b1)